src/main/resources/connectionParameters.properties

```diff
@@ -15,6 +17,11 @@ testOnCdf=false
 cdfUsername=dummy
 cdfPassword=dummy
 
+testByoid=false
+byoidProvider=test-provider
```

Comment:
Where will the actual byoidProvider be provided? test-provider must be a placeholder